llama-index-core/llama_index/core/schema.py

```diff
@@ -483,10 +670,6 @@ def node_info(self) -> Dict[str, Any]:
         return self.get_node_info()
 
 
-# TODO: legacy backport of old Node class
-Node = TextNode
```

Comment:
I wonder how many users were using this legacy backport 😅 Looks like a few tests were using it

Reply:
I searched the codebase and fixed all I could find, it's indeed breaking if anybody was relying on the alias but I saw `legacy` and took the chance :)